tests/test_models.py

```diff
@@ -860,33 +860,40 @@ def test_flatten_messages_as_text_for_all_models(
     [
         # Unsupported base models
         ("o3", False),
+        ("o3-mini", False),
```

Comment:
"o3-mini" (as others, like "o4" and "o4-turbo") was before True, and now it is False. Why? Which value is the right one?